ansible/memcached-operator/README.md

```diff
@@ -1,46 +1,105 @@
-# Memcached Ansible Operator
+# Memcached Operator
 
-## Quickstart
+## Overview
 
-1. Start [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) (e.g. `minikube start`)
+This Memcached operator is a simple example operator for the [Operator SDK][operator_sdk] and includes some basic end-to-end tests.
 
-    We will be using the `default` namespace, and the `default` account
+## Prerequisites
 
-1. Deploy the Memcached Ansible Operator
+- [dep][dep_tool] version v0.5.0+.
+- [go][go_tool] version v1.12+.
+- [docker][docker_tool] version 17.03+
+- [kubectl][kubectl_tool] v1.11.3+
+- [operator-sdk][operator_install]
+- Access to a Kubernetes v1.11.3+ cluster
 
-    * Create the Memcached Ansible Operator RBAC and Service Account files
-        ```bash
-        $ kubectl create -f deploy/service_account.yaml
-        $ kubectl create -f deploy/role.yaml
-        $ kubectl create -f deploy/role_binding.yaml
-        ```
+## Getting Started
 
-    * Create the Memcached Ansible Operator Custom Resource Definitions (CRD)
-        ```bash
-        kubectl create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
-        ```
-    * Deploy the Memcached Ansible Operator
-        ```bash
-        kubectl create -f deploy/operator.yaml
-        ```
+### Cloning the repository
 
-1. Create the Memcached Ansible Operator Custom Resource (CR)
+Checkout this Memcached Operator repository
 
-    ```bash
-    kubectl create -f deploy/crds/cache_v1alpha1_memcached_cr.yaml
-    ```
+```
+$ mkdir $GOPATH/src/github.com/operator-framework
```

Comment:
I'd avoid mention of the `$GOPATH` since you don't actually need `go` to use an Ansible-based operator

Reply:
👍